### PR TITLE
Add max_token_length to whitespace analyzer 

### PR DESCRIPTION
**Pull request by mxm**
Contains two commits. 1) Add the parameter 2) Adjust parameter description for other tokenizers.

### CODE REVIEW

File: blackbox/docs/general/ddl/analyzers.rst [hunk 1]
```diff
@@ -89,8 +89,8 @@ stopwords
     Defaults to the english stop words.
 
 max_token_length
-    The maximum token length. If a token is seen that exceeds this length then
-    it is discarded. Defaults to ``255``.
+    The maximum token length. If a token exceeds this length it is split in
+    max_token_length intervals. Defaults to ``255``.
```
Comment (chaudum) [inline blackbox/docs/general/ddl/analyzers.rst]: ~~intervals~~ chunks

File: blackbox/docs/general/ddl/analyzers.rst [hunk 2]
```diff
@@ -285,8 +285,8 @@ Standard Annex #29.
 .. rubric:: Parameters
 
 max_token_length
-    The maximum token length. If a token is seen that exceeds this length then
-    it is discarded. Defaults to ``255``.
+    The maximum token length. If a token exceeds this length it is split in
+    max_token_length intervals. Defaults to ``255``.
```
Comment (chaudum) [inline blackbox/docs/general/ddl/analyzers.rst]: ~~intervals~~ chunks

File: blackbox/docs/general/ddl/analyzers.rst [hunk 3]
```diff
@@ -304,8 +304,8 @@ languages other than English.
 .. rubric:: Parameters
 
 max_token_length
-    The maximum token length. If a token is seen that exceeds this length then
-    it is split at max_token_length intervals. Defaults to ``255``.
+    The maximum token length. If a token exceeds this length it is split in
+    max_token_length intervals. Defaults to ``255``.
```
Comment (chaudum) [inline blackbox/docs/general/ddl/analyzers.rst]: ~~intervals~~ chunks

File: blackbox/docs/general/ddl/analyzers.rst [hunk 4]
```diff
@@ -346,6 +346,12 @@ Witespace Tokenizer
 
 The ``whitespace`` tokenizer splits text at whitespace.
 
+.. rubric:: Parameters
+
+max_token_length
+    The maximum token length. If a token exceeds this length it is split in
+    max_token_length intervals. Defaults to ``255``.
```
Comment (chaudum) [inline blackbox/docs/general/ddl/analyzers.rst]: ~~intervals~~ chunks

File: blackbox/docs/general/ddl/analyzers.rst [hunk 5]
```diff
@@ -359,8 +365,8 @@ tokenizes emails and urls as single tokens.
 .. rubric:: Parameters
 
 max_token_length
-    The maximum token length. If a token is seen that exceeds this length then
-    it is discarded. Defaults to ``255``.
+    The maximum token length. If a token exceeds this length it is split in
+    max_token_length intervals. Defaults to ``255``.
```
Comment (chaudum) [inline blackbox/docs/general/ddl/analyzers.rst]: ~~intervals~~ chunks

**Comment by mxm** [comment]
@chaudum Addressed your comments.

**Comment by mxm** [comment]
Sorry, merged the fixups commits on accident 😞  Will never use that evil merge button again.

**Pull request merged**